src/groovy/mama/ng/scheduler/CronDefinition.groovy

```diff
@@ -0,0 +1,68 @@
+package mama.ng.scheduler
+
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+class CronDefinition {
+
+    List<Integer> minutes
+    List<Integer> hours
+    List<Integer> daysOfMonth
+    List<Integer> months
+    List<Integer> daysOfWeek
+
+    int minute = 1
+    int hour = 7
+    int dayOfMonth = 17
+    int month = 29
+    int dayOfWeek = 39
+
+    final static String REGEX = '^(((([1-5]?[0-9])(\\,|\\-))*([1-5]?[0-9]))|\\*)\\ ' + //min (0 - 59)
```

Comment:
There is no way of doing named groups in Groovy, so I have done what I can here. Better suggestions are welcome if you have any.